modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/version/DefaultVersionWithScopesTestCase.java

```diff
@@ -19,52 +19,43 @@
 package org.wso2.am.integration.tests.version;
 
 import org.apache.axis2.AxisFault;
+import org.apache.commons.io.IOUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.json.JSONObject;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
+import org.wso2.am.integration.clients.store.api.v1.dto.*;
```

Comment:
Lets remove the * imports

Reply:
Fixed in https://github.com/wso2/product-apim/pull/7429